### DISCUSSION TITLE
Welcome to RP2040-Decoder Discussions!

### DISCUSSION THREAD

**Discussion by gab-k** [gab-k/RP2040-Decoder#3] [Announcements]
### Discussions
 Feel free to ask questions, suggest ideas or share your experience.
If you want to contact me privately feel free to write me an email: gabriel@koppenstein.com

**Comment by hinds44** [comment]
I've just ordered a bunch of these PCBs and will be trying them out on an n gauge layout once they are assembled

**Comment by fmfreitag** [comment]
What is type R4 component, because "DNF" I not found on jclpcb list.

**Comment by HughRey** [comment]
Just getting started with modelling in n guage. As I've been writing embedded c for 35 years, started on Z80, I hope to get involved. Not sure if, or when, I can contribute but very keen to follow this project. I'm based in Wales. Is there an outlet selling fully assembled boards?

**Reply by gab-k** [reply]
I'm glad you like it. Im afraid you would have to order the PCBs yourself. I wrote something about where to get the board in the getting started section of the wiki.
